SignalServiceKit/src/Messages/Reactions/ReactionManager.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 @objc(OWSReactionManager)
 public class ReactionManager: NSObject {
 
-    public static let emojiSet = ["❤️", "👍", "👎", "😂", "😮", "😢"]
+    public static let emojiSet = ["❤️", "👍️", "👎️", "😂", "😮", "😢"]
```

Comment:
What changed here? Some emoji can have a textual representation or an "emoji"/picture presentation, this is due to some emoticons predating emoji. Which one is used can be decided using a [variant selector](https://character.construction/emoji-vs-text). In the old hardcoded emoji, no variant selector was included, so the display of the emoji was system dependant. On macOS this often just defaults to the emoji/picture presentation. The EmojiBase dataset includes the emoji variant selector on its emoji therefore these hardcoded emoji must be changed to match. And in this case I think it is more correct to use the emoji version anyways, as we never want the textural representation. 

Example showing the difference in codepoints from the Swift REPL:
```
  8> let oldEmoji = "👍"
oldEmoji: String = "👍"
  9> let newEmoji = "👍️"
newEmoji: String = "👍️"
 10> print(Array(oldEmoji.unicodeScalars))
["\u{0001F44D}"]
 11> print(Array(newEmoji.unicodeScalars))
["\u{0001F44D}", "\u{FE0F}"]
```